test/libsolidity/semanticTests/various/address_code.sol

```diff
@@ -12,8 +12,6 @@ contract C {
     function g() public view returns (uint) { return address(0).code.length; }
     function h() public view returns (uint) { return address(1).code.length; }
 }
-// ====
-// compileViaYul: false
```

Comment:
So no ``also`` here? Is the code shorter in IR codegen or what happens here?

Reply:
This is missing @mijovic 's PR that implements getters for `bytes`